flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/serialization/EventSerializer.java

```diff
@@ -318,13 +295,9 @@ public static AbstractEvent fromBuffer(Buffer buffer, ClassLoader classLoader) t
 	 *
 	 * @param buffer the buffer to peak into
 	 * @param eventClass the expected class of the event type
-	 * @param classLoader the class loader to use for custom event classes
 	 * @return whether the event class of the <tt>buffer</tt> matches the given <tt>eventClass</tt>
 	 */
-	public static boolean isEvent(final Buffer buffer,
-		final Class<?> eventClass,
-		final ClassLoader classLoader) throws IOException {
-		return !buffer.isBuffer() &&
-			isEvent(buffer.getNioBufferReadable(), eventClass, classLoader);
+	public static boolean isEvent(Buffer buffer, Class<?> eventClass) throws IOException {
+		return !buffer.isBuffer() && isEvent(buffer.getNioBufferReadable(), eventClass);
```

Comment:
similar here: add a comment, that checking for custom events is not supported anymore